pkgs/applications/terminal-emulators/rio/default.nix

```diff
@@ -123,7 +123,7 @@ rustPlatform.buildRustPackage rec {
     license = lib.licenses.mit;
     maintainers = with lib.maintainers; [ tornax otavio oluceps ];
     platforms = lib.platforms.unix;
-    changelog = "https://github.com/raphamorim/rio/blob/v${version}/CHANGELOG.md";
+    changelog = "https://github.com/raphamorim/rio/blob/v${version}/docs/releases.md";
```

Comment:
Wait, shouldn't that be https://github.com/raphamorim/rio/blob/main/docs/docs/releases.md ?

Reply:
I can't open up https://github.com/raphamorim/rio/blob/v0.1.10/docs/releases.md for example